hoptek-labs/python-cli-skeleton-app
# Introduce linting

Required by #3 

Part of epic #1


This PR introduces `black` as linter/formatter for the project.

#### This PR includes
  - `black` as linter/formatter for the project
  - a new `Makefile` target to run linting
  - source files and test files formatted using `black`

#### How to test PR changes locally
Assuming you have already setup this project on your local machine (see `README.md`).
  - Run `make lint`
